docs/entities/nft-events/nft-event.schema.json

```diff
@@ -22,16 +21,26 @@
       "type": "string"
     },
     "value": {
-      "type": "string"
+      "type": "object",
+      "required": ["hex", "repr"],
+      "description": "Identifier of the NFT",
+      "additionalProperties": false,
+      "properties": {
+        "hex": {
+          "type": "string",
+          "description": "Hex string representing the identifier of the NFT"
```

Comment:
"Hex string serialized Clarity value representing the identifier of the NFT" 
?